doc/config.rst

```diff
@@ -294,13 +294,20 @@ General configuration
 
 .. confval:: numfig_format
 
-   A dictionary mapping ``'figure'``, ``'table'`` and ``'code-block'`` to
-   strings that are used for format of figure numbers. Default is to use
-   ``'Fig. %s'`` for ``'figure'``, ``'Table %s'`` for ``'table'`` and
-   ``'Listing %s'`` for ``'code-block'``.
+   A dictionary mapping ``'figure'``, ``'table'``, ``'code-block'`` and
+   ``'section'`` to strings that are used for format of figure numbers.
+   As a special character, `%s` and `{number}` will be replaced to figure
```

Comment:
Hmm, it might be a bug...
Could you file it as a new issue?
I will take it later.